_includes/v22.2/ui/active-statement-executions.md

```diff
@@ -0,0 +1,38 @@
+
+## Active Executions table
+
+Click <img src="{{ 'images/common/ui-columns-button.png' | relative_url }}" alt="Column selector" /> to select the columns to display in the table.
+
+Column | Description
+-----|------------
+Statement Execution ID | The execution ID of the statement execution.
+Statement Execution | The SQL statement that was executed.
+Status | The status of the execution: Preparing, Waiting, or Executing.
+Start Time | The time the execution started.
+Time Spent Waiting | The time the execution spent waiting for access to resources.
```

Comment:
Can we be more explicit and call out lock contention?

The duration the execution spent waiting and experiencing lock contention.

https://www.cockroachlabs.com/docs/v22.1/performance-best-practices-overview#transaction-contention